stable/dex-k8s-authenticator/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for dex-k8s-authenticator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: mintel/dex-k8s-authenticator
+  tag: latest
+  pullPolicy: Always
```

Comment:
not really sure why do we need that if it is not `latest`, and latest is not the best option either.